docs/content/manpages/geowave-cpindex.txt

```diff
@@ -0,0 +1,36 @@
+//:geowave-cpindex(1)
+//:================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave config cpindex - copies and modifies an existing index configuration
+
+SYNOPSIS
+//:--------
+
+*geowave config cpindex* [options] <name> <new name>
+
+DESCRIPTION
+//:-----------
+
+The geowave config cpindex operator will copy and modify an existing index configuration
```

Comment:
Maybe a comment here saying that you can actually override values as you copy, like cpstore old new --gwNamespace new_namespace